charts/shoot-core/charts/csi-packet/templates/controller.yaml

```diff
@@ -0,0 +1,88 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-packet-pd
+  labels:
```

Comment:
addon-manager label is missing

Reply:
And for most of the other resources. Adding now.